code/modules/mob/living/carbon/human/species_types/monkeys.dm

```diff
@@ -0,0 +1,94 @@
+/datum/species/monkey
+	name = "Monkey"
+	id = "monkey"
+	say_mod = "chimpers"
+	attack_verb = "bite"
+	attack_sound = 'sound/weapons/bite.ogg'
+	miss_sound = 'sound/weapons/bite.ogg'
+	mutant_organs = list(/obj/item/organ/tail/monkey)
+	mutant_bodyparts = list("tail_monkey" = "Monkey")
+	skinned_type = /obj/item/stack/sheet/animalhide/monkey
+	meat = /obj/item/food/meat/slab/monkey
+	knife_butcher_results = list(/obj/item/food/meat/slab/monkey = 5, /obj/item/stack/sheet/animalhide/monkey = 1)
+	species_traits = list(HAS_FLESH,HAS_BONE,NO_UNDERWEAR,LIPS,NOEYESPRITES,NOBLOODOVERLAY,NOTRANSSTING, NOAUGMENTS)
+	inherent_traits = list(TRAIT_MONKEYLIKE)
+	no_equip = list(ITEM_SLOT_EARS, ITEM_SLOT_EYES, ITEM_SLOT_OCLOTHING, ITEM_SLOT_GLOVES, ITEM_SLOT_FEET, ITEM_SLOT_ICLOTHING, ITEM_SLOT_SUITSTORE)
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_PRIDE | MIRROR_MAGIC | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
```

Comment:
Probably want to remove race_swap here since these are meant to be harmless iirc.